share/change.py

```diff
@@ -298,7 +298,7 @@ def __init__(self, graph, id, type, attrs, namespace=None):
         self.namespace = namespace
 
         if not self.is_blank:
-            self.instance = IDObfuscator.load(self.id, None)
+            self.instance = IDObfuscator.resolve(self.id)
```

Comment:
Hmm... one way or another I don't think both should exist. However whatever name we go with should also take a `canonical=False` (default to `True`) that will just load the non-canonical record.

I'm partial to `load` just because its fewer characters that I have to type. 🍆